script/release

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
+
+# check that gh is installed and logged in 
+
+gh auth status -h github.com
```

Comment:
Since this is before the `set -e`, if it fails it won't stop the script from running.